code/modules/crew_objectives/security_objectives.dm

```diff
@@ -28,3 +28,14 @@
 		if(!(M.mind.assigned_role in GLOB.security_positions) && istype(get_area(M), /area/security/prison)) //there's no list of incarcerated players, so we just assume any non-security people in prison are prisoners, and assume that any security people aren't prisoners
 			return FALSE
 	return TRUE
+
+/datum/objective/crew/justicemed
+	explanation_text = "Ensure there are no dead bodies in the security wing when the shit ends."
```

Comment:
"when the shit ends."

Basado

Reply:
Sometimes the funnies write themselves accidentally.